src/BizHawk.Client.EmuHawk/tools/TAStudio/TAStudio.MenuItems.cs

```diff
@@ -1355,19 +1355,6 @@ private void SetUpToolStripColumns()
 			TasView.AllColumns.ColumnsChanged();
 		}
 
-		// ReSharper disable once UnusedMember.Local
-		[RestoreDefaults]
-		private void RestoreDefaults()
-		{
-			TasView.AllColumns.Clear();
-			SetUpColumns();
-			TasView.Refresh();
-			CurrentTasMovie.FlagChanges();
-
-			MainVertialSplit.SplitterDistance = _defaultMainSplitDistance;
-			BranchesMarkersSplit.SplitterDistance = _defaultBranchMarkerSplitDistance;
-		}
-

```

Comment:
Removing this will 99% break stuff. In general I believe it's not good for the format command to remove unused code, as there are multiple valid reasons for why stuff may be "unused" but should still stay.
Is there a way to disable that specifically?

Reply:
definitely and I think I what I would do here if we go that way would be to use a `#pragma` directive to disable the rule for those lines in case other people want to have the analyzers on in their IDE or if it gets run in CI/pre commit hooks in the future

Reply:
There should be some way to mark functions as used with attributes like `RestoreDefaults` if I recall correctly, perhaps we should look into that.

Reply:
It looks like it's hitting IDE0051 removing unused private members and that warning still hits when marked with the attribute per other issues

https://github.com/dotnet/roslyn/issues/34986

I think you have to wrap it with the pragma directive to ignore IDE0051

Reply:
I've updated the changes with IDE0051 warnings pragma'ed